crates/qt-build-utils/src/lib.rs

```diff
@@ -1067,7 +1097,22 @@ Q_IMPORT_PLUGIN({plugin_class_name});
             );
         }
 
-        output_path
+        let qt_6_5 = SemVer {
+            major: 6,
+            minor: 5,
+            ..SemVer::default()
+        };
+        let init_header = if self.version >= qt_6_5 {
+            // With Qt6.5 the Q_INIT_RESOURCE macro is in the QtResource header
+            "QtResource"
+        } else {
+            "QDir"
```

Comment:
These could both be `QtCore/QDir` and `QtCore/QtResource` i think, to make it obvious these are coming from the Qt Core module

Reply:
Done